AdminWebsite/AdminWebsite/ClientApp/src/app/booking/add-participant/add-participant.component.ts

```diff
@@ -197,52 +159,8 @@ export class AddParticipantComponent extends BookingBaseComponent implements OnI
     }
 
     initializeForm() {
-        this.role = new FormControl(this.constants.PleaseSelect, [
-            Validators.required,
-            Validators.pattern(this.constants.PleaseSelectPattern)
-        ]);
-        this.party = new FormControl(this.constants.PleaseSelect, [
-            Validators.required,
-            Validators.pattern(this.constants.PleaseSelectPattern)
-        ]);
-        this.title = new FormControl(this.constants.PleaseSelect);
-        this.firstName = new FormControl('', [
-            Validators.required,
-            Validators.pattern(Constants.TextInputPatternName),
-            Validators.maxLength(255)
-        ]);
-        this.lastName = new FormControl('', [
-            Validators.required,
-            Validators.pattern(Constants.TextInputPatternName),
-            Validators.maxLength(255)
-        ]);
-        this.phone = new FormControl('', [Validators.required, Validators.pattern(Constants.PhonePattern)]);
-        this.displayName = new FormControl('', [
-            Validators.required,
-            Validators.pattern(Constants.TextInputPatternName),
-            Validators.maxLength(255)
-        ]);
-        this.companyName = new FormControl('');
-        this.companyNameIndividual = new FormControl('', [Validators.pattern(Constants.TextInputPattern), Validators.maxLength(255)]);
-        this.representing = new FormControl('', [Validators.pattern(Constants.TextInputPattern), Validators.maxLength(255)]);
-        this.interpreterFor = new FormControl(this.constants.PleaseSelect, [
-            Validators.required,
-            Validators.pattern(this.constants.PleaseSelectPattern)
-        ]);
-
-        this.form = new FormGroup({
-            role: this.role,
-            party: this.party,
-            title: this.title,
-            firstName: this.firstName,
-            lastName: this.lastName,
-            phone: this.phone,
-            displayName: this.displayName,
-            companyName: this.companyName,
-            companyNameIndividual: this.companyNameIndividual,
-            representing: this.representing,
-            interpreterFor: this.interpreterFor
-        });
+        this.initialiseForm();
```

Comment:
These names are confusing and should be changed.

Reply:
Which names are confusing?

Reply:
I agree. Party for example is very confusing, but these field names are used across multiple files and renaming them isn't trivial without testing multiple components and making sure they don't break, so I left them as is. 

There is a tech debt created to refactor this further: https://tools.hmcts.net/jira/browse/VIH-8127
Maybe you can add to that regarding changing the field names?

Reply:
I mean having an initializeForm and an initialiseForm

Reply:
Ah yeah I found that weird too. The original had the American spelling. Can you imagine! We are doing work for Her Majesty using American spelling! Surely that's treason!